package.json

```diff
@@ -28,16 +28,21 @@
     "partial-lint": "git diff --name-only master... | grep js$ | xargs ./node_modules/.bin/eslint",
     "check": "npm run lint && npm run test",
     "tempcheck": "npm run partial-lint && npm run test",
-    "test": "jest"
+    "test": "jest",
+    "watch": "jest --watch"
   },
   "author": "",
   "license": "ISC",
   "jest": {
-    "preset": "jest-expo"
+    "preset": "jest-expo",
+      "transformIgnorePatterns": [
+        "node_modules/(?!native-base-shoutem-theme|react-native-easy-grid|native-base|react-native|react-navigation|expo|@expo)"
```

Comment:
Can you explain the changes you have made here?

Reply:
native-base by default is in ES6 so it doesn't need to be transformed. 

GeekyAnts/NativeBase#713

Reply:
Sorry I didn't realise this was actually mentioned in your pull request description.